server/src/index.ts

```diff
@@ -33,4 +33,23 @@ function mongodb_connect() {
 }
 
 mongodb_connect();
+
+app.post('/login', (req: Request, res: Response) => {
```

Comment:
`app` is not defined anyway in the program.  Typescript is throwing an error because of this:
https://github.com/pogi7/luminosityleds/actions/runs/3805483361/jobs/6473461441#step:5:23